### PR TITLE
update the parameter for the wordtag

### PR DESCRIPTION
**Pull request by wawltor**
### PR types
Function optimization

### PR changes
 Others 

### Description
update the parameter for the wordtag

### CODE REVIEW

File: paddlenlp/taskflow/knowledge_mining.py [hunk 1]
```diff
@@ -167,12 +167,12 @@ class WordTagTask(Task):
...
             "model_state": [
-                "https://bj.bcebos.com/paddlenlp/taskflow/knowledge_mining/wordtag/model_state.pdparams",
-                "12685d1d84c09fb851b6c1541af1146e"
+                "https://bj.bcebos.com/paddlenlp/taskflow/knowledge_mining/wordtag_v1.1/model_state.pdparams",
+                "a93e4074c9440cc3f1d4d29e54cf766d"
             ],
             "model_config": [
-                "https://bj.bcebos.com/paddlenlp/taskflow/knowledge_mining/wordtag/model_config.json",
-                "aa47cdf7c270943a24495bd5ff59dc00"
+                "https://paddlenlp.bj.bcebos.com/taskflow/knowledge_mining/wordtag_v1.1/model_config.json",
```
Comment (linjieccc) [inline paddlenlp/taskflow/knowledge_mining.py]: paddlenlp.bj.bcebos.com -> bj.bcebos.com/paddlenlp
bos链接前缀统一

File: paddlenlp/taskflow/knowledge_mining.py [hunk 2]
```diff
@@ -183,8 +183,8 @@ class WordTagTask(Task):
                 "a0efe723f84cf90540ac727be5b62e59"
             ],
             "tags": [
-                "https://bj.bcebos.com/paddlenlp/taskflow/knowledge_mining/wordtag/tags.txt",
-                "87db06ae6ca42565157045ab3e9a996f"
+                "https://paddlenlp.bj.bcebos.com/taskflow/knowledge_mining/wordtag_v1.1/tags.txt",
+                "f33feedd01d478b03bac81be19b48d00"
```
Comment (linjieccc) [inline paddlenlp/taskflow/knowledge_mining.py:187]: 同上

Comment (wawltor) [reply paddlenlp/taskflow/knowledge_mining.py:187]: done

File: paddlenlp/taskflow/knowledge_mining.py (hunk 1 above)
Comment (wawltor) [reply paddlenlp/taskflow/knowledge_mining.py]: > paddlenlp.bj.bcebos.com -> bj.bcebos.com/paddlenlp bos链接前缀统一

done

**Review by ZeyuChen — APPROVED** [review]

**Review by linjieccc — APPROVED** [review]
LGTM

**Pull request merged**